app/views/layouts/application.html.erb

```diff
@@ -9,6 +9,8 @@
   </head>
 
   <body>
+	<p class="notice"><%= notice %></p>
```

Comment:
A note, Devise requested that we have those in for flash messages after login/logout. I'll put them in the internal site when we have it.